### PR TITLE
update contributing.md to point to sonarcloud rules

### PR DESCRIPTION
**Pull request by adamkorynta**
## Problem Description

SonarCloud rules are somewhat buried within the OpenDCS organization.

## Solution

Adding a link to the contributing.md to point to where the rules are configured.

## how you tested the change

Tested link.

## Where the following done:

- [X] Was relevant documentation updated?

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -123,9 +123,11 @@ The current java target is java 8. Please make sure you do not add java 9+ featu
 
 Now you're ready to [clone the repository](https://help.github.com/articles/cloning-a-repository/) and start looking at things. If you are going to submit any changes you make, please read the Submitting changes section below.
 
-### Code Style
+### Coding Practices
 
-TODO: add checkstyle to gradle build
+Code style rules as well as other static analysis rules are setup for the [OpenDCS Organization on SonarCloud](https://sonarcloud.io/organizations/opendcs/quality_profiles/show?name=CWMS_ATO&language=java).
```
Comment (MikeNeilson) [inline CONTRIBUTING.md:128]: I assume we're going to change the quality profile to something "OpenDCS" and adjust the rules accordingly at some point?

**Review by MikeNeilson — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (adamkorynta) [reply CONTRIBUTING.md]: Updated to just "OpenDCS". We can always extend/rename later to create an "OpenDCS REST API" in the future as needed

**Review by MikeNeilson — COMMENTED** [review]
You can delete the .DS_store files, I think.
If those folders are empty and need to stay then something like ".keep" is better.
But I'm pretty sure .DS_store is from CSV or SVN

**Comment by adamkorynta** [comment]
.DS_store are MacOS only. They were excluded in the .gitignore so they didn't get purged when I reorganized the directories.

**Pull request merged**